client/src/components/Footer.js

```diff
@@ -0,0 +1,98 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from '@emotion/styled';
+import ListsIcon from '../assets/lists-icon.svg';
+import ListsIconClicked from '../assets/lists-clicked-icon.svg';
+import PopularIcon from '../assets/popular-icon.svg';
+import PopularIconClicked from '../assets/popular-clicked-icon.svg';
+import RecsIcon from '../assets/recs-icon.svg';
+import RecsIconClicked from '../assets/recs-clicked-icon.svg';
+
+const FooterContainer = styled.div`
```

Comment:
I would call this component rather "Tab bar", "Bottom nav" or something like this, because it's not really a footer!